packages/terra-slide-panel-manager/src/SlidePanelManager.jsx

```diff
@@ -15,6 +15,11 @@ const propTypes = {
    * disclosure capabilities through the DisclosureManger's context API.
    */
   children: PropTypes.node,
+  /**
+   * Sets the heading level of the disclosed panel. One of `1`, `2`, `3`, `4`, `5`, `6`. This helps screen readers to announce appropriate heading levels.
+   * Changing 'level' will not visually change the style of the content.
+   */
+  level: PropTypes.oneOf([1, 2, 3, 4, 5, 6]),
```

Comment:
Since `level` is set on the SlidePanelManager, if a consumer wanted to disclose a slide panel of level 2 and then disclose a second slide panel of level 3 from within the level 2 one, would they be able to do that?

Reply:
The header is shared across multiple slides, so a consumer would not be able to do that, only one level is allowed

Reply:
That might be an issue. BRB bugging some people.

Reply:
Talked to Scott and he said it's cool!